tests/integration/integration_go_test.go

```diff
@@ -1221,13 +1221,41 @@ func TestParameterizedGo(t *testing.T) {
 		PrePrepareProject: func(info *engine.Projinfo) error {
 			e := ptesting.NewEnvironment(t)
 			e.CWD = info.Root
+
+			// Overwrite the main program with the actual test program that uses the generated SDK.
```

Comment:
Could you add a comment on why we do this dance with the `.txt` file? (helping dependabot)